src/test/java/com/amazon/opendistroforelasticsearch/sql/unittest/DateFormatTest.java

```diff
@@ -34,6 +34,7 @@
 import org.elasticsearch.index.query.RangeQueryBuilder;
 import org.hamcrest.*;
```

Comment:
could you please get rid of * import?

Reply:
The reges import .*\.\* shows 30 matches in 17 files, I think it's better to do that cleanup in separate PR